internal/venice-common/src/main/java/com/linkedin/venice/blobtransfer/client/P2PFileTransferClientHandler.java

```diff
@@ -117,7 +119,7 @@ protected void channelRead0(ChannelHandlerContext ctx, HttpObject msg) throws Ex
 
       if (content instanceof DefaultLastHttpContent) {
         // End of a single file transfer
-        LOGGER.debug("A file {} received successfully for {}", fileName, payload.getFullResourceName());
+        LOGGER.info("A file {} received successfully for {}", fileName, payload.getFullResourceName());
```

Comment:
I'd like to keep it debug for the same reason above.

Reply:
back to debug in the latest iteration